### PR TITLE
Updating info on instruction

### PR DESCRIPTION
**Pull request by PeterBell**
Killing outdated copy and providing detailed information on instructional support - thoughts @jmburges ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,51 @@
...
+
+You can host your own study groups to connect with and learn with other students. However, we also run over 40 study groups a week that are "Flatiron hosted" to help you to learn to code.
+
+There are seven different types of study groups:
+- **Orientation**: Optional advice and information on getting started with the program. Usually on Wednesdays every week.
+- **Motivational** (Thoughtful Thursday, Coders Unite, etc): The objective of these is to allow students to connect and chat about their experiences of being on learn and to motivate each other.
+- **Office Hours**: A chance to connect with an instructor to ask any technical questions you might have
+- **Portfolio Project Prep**: A chance to talk with an instructor about any questions you might have about designing or building your portfolio project review app
+- **Lab Specific**: A combination of code along and a description of the underlying concepts.
+- **Technical**: A study group that focuses on a key concept rather than a specific lab.
```
Comment (cernanb) [inline README.md]: Can you update `Technical` to `Live Lecture`. Study groups of this type we name as a Live Lecture. Updated this on the Learn TA repo as well.

**Comment by kvignali** [comment]
@PeterBell I added the IDE download button to lesson page for now (in case anyone slips through while you're improving it). You can grab the new button from the lesson if you want to add that to your update.

**Comment by PeterBell** [comment]
Hey @kvignali conceptually I'm open to doing that, but I'm not sure how to add a button to markdown in a way that it'll render correctly in learn. Any idea how to do that>? In the meantime merging in what I've got, but LMK - happy to make the change!

**Comment by PeterBell** [comment]
Hey @kvignali doh - I see you added it already - thanks so much!!!

**Pull request merged**